ase/src/main/java/com/ase/ase/dao/AustriaRepository.java

```diff
@@ -3,6 +3,15 @@
 import com.ase.ase.entities.AgeDistribution;
 import com.ase.ase.entities.Austria;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
 
 public interface AustriaRepository extends JpaRepository<Austria, Long> {
+
+    @Query("Select a.newCases, a.time from Austria a")
+    List<Object[]> findAllNewCases();
+
+    @Query("Select a.sumCases-a.sumCured-a.sumCured, a.time from Austria a")
```

Comment:
You substract the cured two times. Must be -cured -dead
If this is changed, branch can be merged :)